package.json

```diff
@@ -62,6 +64,9 @@
     "husky": "^7.0.4",
     "lint-staged": "^12.1.2",
     "next": "^12.0.7",
+    "nextv9": "npm:next@9.0.0",
+    "nextv10": "npm:next@10.0.0",
+    "nextv11": "npm:next@11.0.0",
```

Comment:
@RyanClementsHax I know there was a minor version that changed the location of RouterContext

Reply:
You're right. It was 11.1 that had those changes. This package is a dev dependency just for typings though. Do you think I should also include 11.1?

Reply:
@RyanClementsHax depends, you could probably just get away with telling them to upgrade.

Reply:
That's a good point